LICENSE.md

```diff
@@ -10,16 +10,16 @@ The Original Version (as defined in the SIL Open Font License, Version 1.1) rema
 
 This Modified Version (Public Sans) contains both software under the SIL Open Font License, Version 1.1 and software modifications by GSA released as CC0. It is a “joint work” made of the original software and modifications combined into a single work.
 
-**In practice, users of this Modified Version (Public Sans) should use Public Sans according to the terms of the SIL Open Font License, Version 1.1.** This is because this font is a combination of work subject to copyright and work not subject to copyright, so the more restrictive requirements apply to using the combined work.
+**In practice, users of this Modified Version (Public Sans) should use Public Sans according to the terms of the SIL Open Font License, Version 1.1, below.** This is because this font is a combination of work subject to copyright and work not subject to copyright, so the more restrictive requirements apply to using the combined work.
 
 ## License of project USWDS’s Modified Version is based on
 
 - Libre Franklin is licensed under the SIL Open Font License, Version 1.1 (<http://scripts.sil.org/OFL>)
-- To view the copyright and specific terms and conditions please refer to [OFL.txt](https://github.com/impallari/Libre-Franklin/blob/master/OFL.txt)
+- To view the copyright and specific terms and conditions of Libre Franklin, please refer to [OFL.txt](https://github.com/impallari/Libre-Franklin/blob/master/OFL.txt)
 
-## Libre Franklin SIL Open Font License, Version 1.1
+## SIL Open Font License, Version 1.1
 
-Copyright (c) 2015, Impallari Type (www.impallari.com)
```

Comment:
@thisisdano this and the change to OFL may need to be rebased out to another branch